nefertari_sqla/documents.py

```diff
@@ -167,14 +167,20 @@ def get_es_mapping(cls, _depth=None, types_map=None):
         mapper = class_mapper(cls)
         columns = {c.name: c for c in mapper.columns}
         relationships = {r.key: r for r in mapper.relationships}
-
         for name, column in columns.items():
             column_type = column.type
             if isinstance(column_type, types.ChoiceArray):
                 column_type = column_type.impl.item_type
             column_type = type(column_type)
+
             if column_type not in types_map:
                 continue
+
+            if hasattr(column, '_custom_analyzer') and getattr(column, '_custom_analyzer'):
```

Comment:
You can do without `hasattr` if you specify a default value to `getattr`:
```python
if getattr(column, '_custom_analyzer', False):
```
See https://docs.python.org/3/library/functions.html#getattr